python/src/nanoarrow/_lib.pyx

```diff
@@ -2069,6 +2166,17 @@ cdef class CBufferView:
         else:
             return self._element_size_bits // 8
 
+
+    def __dlpack__(self):
+        # Note: parent offset not applied!
```

Comment:
Happy to add docstrings here but didn't currently as I see docstrings are not used elsewhere and I am guessing they are not checked and therefore avoided?

Reply:
Let's add a docstring anyway! ;) 
(you can just copy what you wrote in pyarrow)